app/models/application.rb

```diff
@@ -64,6 +64,17 @@ class Application < ApplicationRecord
     withdrawn: "withdrawn",
   }
 
+  # eligible_for_funding is solely based on what NPQ app knows
+  # eg school, course etc
+  # here we need to account for previous enrollments too
+  def eligible_for_dfe_funding?
```

Comment:
@cwrw following the same approach as in https://github.com/DFE-Digital/early-careers-framework/pull/4871, we would need to consider two scenarios: with funded place and without funded place. 
But this method is not used yet, so it is unclear to me how we should tackle this.

It is also unclear to me the difference between `eligible_for_dfe_funding` and `eligible_for_funding`. 

- `eligible_for_funding` is calculated in NPQ app based on the applicant responses
- `eligible_for_dfe_funding?` takes into consideration the LP decision about the applicant 

New developers coming to the code might find this confusing, I would aim for a different method name?

What about
- `funded_by_dfe?`
- `requested_for_dfe_funding?`

Reply:
Discussed in person, we will handle it after separation by including this logic into the eligibility calculation.